include/vb_gui.h

```diff
@@ -1,6 +1,7 @@
 #ifndef VB_GUI_H
 #define VB_GUI_H
 
+#include "3ds.h"
```

Comment:
vb_gui defines `waitForInput` that returns a `u32` but doesn't have the `u32` type included yet. (this is showing up because allegro_compat.c implicitly declares waitForInput because it doesn't include vb_gui, including vb_gui in allegro_compat.c changes the import ordering.)